src/Codec/Binary/Bech32/Internal.hs

```diff
@@ -7,7 +7,7 @@
 {-# LANGUAGE UndecidableInstances #-}
 
 -- |
--- Copyright: © 2017 Marko Bencun, 2018-2019 IOHK
+-- Copyright: © 2017 Marko Bencun, 2019-2020 IOHK
```

Comment:
Should that not be `2018-2020` or was the original `2018` incorrect?

Reply:
The original 2018 was incorrect.

IOHK started working on this in 2019, rather than 2018.